src/realm/sort_descriptor.cpp

```diff
@@ -134,6 +134,12 @@ std::unique_ptr<BaseDescriptor> SortDescriptor::clone() const
 
 void SortDescriptor::merge(SortDescriptor&& other, MergeMode mode)
 {
+    if (mode == MergeMode::replace) {
+        m_column_keys = other.m_column_keys;
```

Comment:
These two lines need std::move() to actually move from `other` rather than copying it.

Reply:
The rest of this function makes copies rather than `std::move()`'ing `other`, so I was trying to match those semantics. I guess the r-value qualifier on `other` means this function should always move/consume `other`. I'll fix that up.

Reply:
When it's adding to the existing one rather than replacing it there isn't anything to usefully move since the values are just ints and bools. I'm not really sure why the function previously took a rvalue reference, but at least it's mildly useful to be doing so now?